### ISSUE TITLE
When towny is disabled chat will not work

### ISSUE THREAD

**Issue by RSNFreud** [TownyAdvanced/Towny#4780]
### What steps will reproduce the problem?
1. Disable towny in a world
2. Chat
3. Observe chat not appearing

### What is the expected output?
1. Chat is shown

### What is your Towny version?
0.96.7.0

### What is your spigot/craftbukkit version?
Paper-325 (MC 1.16.4)

### Please use Pastebin.com to link the following files:

**Skipping these files is not an option. It is the easiest way to diagnose an issue ticket properly. If you do skip this step then your ticket will most likely be deleted and you will be asked to resubmit.**
1. Your full server startup from the logs\latest.log : https://pastebin.com/DjZL4W4u
2. Your Towny config.yml (if using MySQL please remove password,) : https://pastebin.com/x3b8Fv7E
3. Your townyperms.yml : https://pastebin.com/29tgcDKK
4. Your log's error : None

(optional) If this is to do with permissions, 
5. Your permissions file : https://luckperms.net/editor/Zzkw4gGMrB

**Comment by LlmDl** [comment]
Update to TownyChat 0.87 and re-test.

**Comment by RSNFreud** [comment]
Solved :)

**Comment by RSNFreud** [comment]
Didn't realise I was in OP. Bug still persists

**Comment by LlmDl** [comment]
You filed a chat related ticket and didn't submit your chatconfig and channels files.

**Comment by RSNFreud** [comment]
ChatConfig.yml: https://pastebin.com/6UqFr880
Channels.yml: https://pastebin.com/c4gyUxwN

**Comment by LlmDl** [comment]
Unable to reproduce. It is likely another plugin causing this, as you have modify_chat false. 

To test I used your chatconfig and channels.yml, TownyChat 0.87, Towny 0.96.7.5, and followed your instructions.

After disabling Towny in the world I was able to see myself chatting. 

After setting modify_chat to false I could still see myself chatting but lost all TownyChat formatting. 

After using your chatconfig and channels files I was able to still see myself chatting with the default MC chat formatting.
